08-rails/routes-controllers.md

```diff
@@ -45,6 +45,77 @@ Let's break down this code:
   - **request method**: `get` in this example
   - **path**: what will be matched with the URL in the HTTP request, `"/books"` in this example
   - **controller and action**: defining the controller and action, split by the `#`. `"books#index"` in this case this would point to the `index` method in the `BooksController` class.
+  
+### Naming Routes 
+
+A `routes.rb` file which lists all the CRUD operations would look like this:
+
+```ruby
+# config/routes.rb
+Rails.application.routes.draw do
+
+  get "/books/new", to: "books#new", as: "new_book"
+  post "/books", to: "books#create"
+
+  get '/books/:id/edit', to: 'books#edit', as: "edit_book"
+  patch '/books/:id', to: 'books#update'
+
+  get "/books", to: "books#index"
+  get "/books/:id", to: "books#show", as: "book"
+
+  delete "/books/:id", to: "books#destroy", as: "delete_books"
```

Comment:
The name for this route should be `delete_book` to match the `rails routes` listing below.